packages/nextjs/next.config.ts

```diff
@@ -9,9 +9,16 @@ const nextConfig: NextConfig = {
   eslint: {
     ignoreDuringBuilds: process.env.NEXT_PUBLIC_IGNORE_BUILD_ERROR === "true",
   },
-  webpack: config => {
+  webpack: (config, { dev }) => {
     config.resolve.fallback = { fs: false, net: false, tls: false };
     config.externals.push("pino-pretty", "lokijs", "encoding");
+    if (dev) {
+      config.watchOptions = {
+        followSymlinks: true,
+      };
+
+      config.snapshot.managedPaths = [];
+    }
```

Comment:
What does this allow?

Reply:
Ohh will remove this later 🙌, we have this when linking package from local repo